catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/NsiliTestCommon.java

```diff
@@ -51,7 +51,7 @@
 import ddf.security.service.SecurityManager;
 import ddf.security.service.SecurityServiceException;
 
-public class NsiliCommonTest {
+public class NsiliTestCommon {
```

Comment:
❓ was the idea just to rename this for unit test file naming convention plus it wouldn't count as a unit test file with 0 unit tests?